src/controllers/MainController.js

```diff
@@ -0,0 +1,67 @@
+const Model = require('../Model/Model');
+const { 
+  readPlayerNum,
+  readRestartEnd,
+  printBallStrike,
+  printBall,
+  printStrike,
+  printNothing,
+  printThreeStrike,
+  exit,
+ } = require('../view/view');
+
+
+class MainController {
+  #model
+
+  mainGameController(){
+    this.#model = new Model()
+    this.sendPlayerNum();
+  }
```

Comment:
어차피 생성할 때 한번 실행해야하는건데

그럴거면 그냥 constructor 쓰세요

```suggestion
  constructor(){
    this.#model = new Model()
    this.sendPlayerNum();
  }
```